ceph-installer-rpm/build/build

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -ex
+
+sudo yum -y install epel-release
+sudo yum -y install fedpkg
+
+# Mock configuration file.
+# TODO: Use the CentOS Storage SIG repository instead of this Fedora Copr repo.
+# (discussion ongoing on the centos-devel list)
+MOCK_CFG=https://fedorapeople.org/~ktdreyer/ceph-installer/ktdreyer-ceph-installer.cfg
+
+# Install this mock configuration file.
+if [ ! -f /etc/mock/$(basename $MOCK_CFG) ]; then
+  # We only refresh the config file in /etc/mock if it does not already exist.
+  # If we were to refresh this file on every run, we would bump the timestamp
+  # each time, and mock would see the timestamp change and invalidate its
+  # cache. By checking the file in /etc/mock first, we can save a bit of build
+  # time here.
+  curl $MOCK_CFG | sudo tee /etc/mock/$(basename $MOCK_CFG)
+fi
+
+# Add the Jenkins slave UID to the mock group.
+sudo usermod -a -G mock $(whoami)
+
+# Attempt the build. If it fails, print the mock logs to STDOUT.
+make rpm || ( tail -n +1 {root,build}.log && exit 1 )
+
+BRANCH=`branch_slash_filter $BRANCH`
+
+# Upload to chacra:
+virtualenv venv
```

Comment:
Could you please use the methods in ``scripts/build_utils.sh`` to make the virtualenv and install chacractl.

An example: https://github.com/ceph/ceph-build/blob/master/ceph-dev-build/build/setup_deb#L105-L111